modules/nw-controlling-dns-pod-placement.adoc

```diff
@@ -23,25 +28,49 @@ As a cluster administrator, you can use a custom node selector to configure the
 .Procedure
 
 * To allow the daemon set for CoreDNS to run on certain nodes, configure a taint and toleration:
+
+. Set a taint on the nodes that you want to control DNS pod placement by entering the following command:
++
+[source,terminal]
+----
+$ oc adm taint nodes <node_name> dns-only=abc:NoExecute
+----
 +
-. Modify the DNS Operator object named `default`:
+Replace `<node_name>` with the actual name of the node.
+
+. Modify the DNS Operator object named `default` to include the corresponding toleration by entering the following command: 
 +
 [source,terminal]
 ----
 $ oc edit dns.operator/default
 ----
-+
-. Specify a taint key and a toleration for the taint:
+
+. Specify a taint key and a toleration for the taint. The following toleration matches the taint set on the nodes.
 +
 [source,yaml]
 ----
  spec:
    nodePlacement:
      tolerations:
      - effect: NoExecute
-       key: "dns-only"
-       operators: Equal
+       key: "dns-only" <1>
+       operator: Equal
        value: abc
-       tolerationSeconds: 3600 <1>
+       tolerationSeconds: 3600 <2>
 ----
-<1> If the taint is `dns-only`, it can be tolerated indefinitely. You can omit `tolerationSeconds`.
+<1> If the `key` field is set to `dns-only`, it can be tolerated indefinitely.
+<2> The `tolerationSeconds` field is optional.
+
+* To specify node placement using a node selector, modify the default DNS Operator:
+
+. Edit the DNS Operator object named `default` to include a node selector:
++
+[source,yaml]
+----
+ spec:
+   nodePlacement:
+     nodeSelector:
+       node-role.kubernetes.io/control-plane: ""
+----
++
+This node selector ensures that the CoreDNS pods run only on control plane nodes.
```

Comment:
```suggestion
. Optional: To specify node placement using a node selector, modify the default DNS Operator:

..
.. Edit the DNS Operator object named `default` to include a node selector:
+
[source,yaml]
----
 spec:
   nodePlacement:
     nodeSelector:    <1>
       node-role.kubernetes.io/control-plane: ""
----
+
<1> This node selector ensures that the CoreDNS pods run only on control plane nodes.
```

Reply:
Just a formatting suggestion, but I think that this should be a numbered step with an `Optional:`. As a bulleted item, it seems sorta orphaned. Also, another callout for the node selector reference.